src/containers/Admin/EditTab/index.tsx

```diff
@@ -0,0 +1,167 @@
+        () =>
+            nearestPlaces
+                .filter(({ type }) => type === 'StopPlace')
+                .map(({ id }) => id),
+        [nearestPlaces],
+    )
+
+    useEffect(() => {
+        const ids = [...newStops, ...nearestStopPlaceIds]
+        if (ids.length) {
+            getStopPlacesWithLines(
+                ids.map((id) => id.replace(/-\d+$/, '')),
+            ).then((resultingStopPlaces) => {
+                setStopPlaces(
+                    resultingStopPlaces.map((s, index) => ({
+                        ...s,
+                        id: ids[index],
+                    })),
+                )
+            })
+        }
+    }, [nearestPlaces, nearestStopPlaceIds, newStops])
+
+    useEffect(() => {
+        const nearestBikeRentalStationIds = nearestPlaces
+            .filter(({ type }) => type === 'BikeRentalStation')
+            .map(({ id }) => id)
+        const ids = [...newStations, ...nearestBikeRentalStationIds]
+        if (ids.length) {
+            service.getBikeRentalStations(ids).then((freshStations) => {
+                const sortedStations = freshStations.sort(
+                    (a: BikeRentalStation, b: BikeRentalStation) =>
+                        a.name.localeCompare(b.name, 'no'),
+                )
+                setStations(sortedStations)
+            })
+        }
+    }, [nearestPlaces, newStations])
+
+    const addNewStop = useCallback(
+        (stopId: string) => {
+            const numberOfDuplicates = [...nearestStopPlaceIds, ...newStops]
+                .map((id) => id.replace(/-\d+$/, ''))
+                .filter((id) => id === stopId).length
+            const id = !numberOfDuplicates
+                ? stopId
+                : `${stopId}-${numberOfDuplicates}`
+            setNewStops([...newStops, id])
+        },
+        [nearestStopPlaceIds, newStops, setNewStops],
+    )
+
+    const addNewStation = useCallback(
+        (stationId: string) => {
+            setNewStations([...newStations, stationId])
+        },
+        [newStations, setNewStations],
+    )
+
+    const modes: Array<{
+        mode: LegMode
+        subMode?: TransportSubmode
+    }> = useMemo(() => {
+        const modesFromStopPlaces = stopPlaces
+            .map((stopPlace) =>
+                stopPlace.lines.map(({ transportMode, transportSubmode }) => ({
+                    mode: transportMode,
+                    subMode: transportSubmode,
+                })),
+            )
+            .reduce((a, b) => [...a, ...b], [])
+            .filter(({ mode }) => isLegMode(mode))
+
+        const uniqModesFromStopPlaces = unique(
+            modesFromStopPlaces,
+            (a, b) => a.mode === b.mode,
+        )
+
+        return stations.length
+            ? [{ mode: 'bicycle' }, ...uniqModesFromStopPlaces]
+            : uniqModesFromStopPlaces
+    }, [stations.length, stopPlaces])
+
+    return (
+        <div>
+            <Heading2>Rediger Innhold</Heading2>
```

Comment:
Skal ha liten "I"

Reply:
Tror også dette egentlig burde være en <Heading1> for accessibility, vet Difi er ganske strenge på sånne elementer https://www.difi.no/fagomrader-og-tjenester/digitalisering-og-samordning/kvalitet-pa-nett/kriteriesett/45-alt-innhold-er-korrekt-kodet